code/_helpers/global_lists.dm

```diff
@@ -93,6 +93,17 @@ var/global/list/string_slot_flags = list(
 		var/datum/sprite_accessory/marking/M = new path()
 		global.body_marking_styles_list[M.name] = M
 
+	// Keybindings
+	for(var/KB in subtypesof(/datum/keybinding))
```

Comment:
If you never actually need duplicate keybdings, it would be preferable to make a `/decl/keybinding`.

Reply:
I still don't know about decl for keybindings. If main philosophy of decl is to make static data holder, keybindings which contain edited data will probably a mistake.